### PR TITLE
Chore/enhancements pismo

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

File: packages/pismo-id/LoginForm.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export const LoginForm = (props: LoginFormProps) => {
                   type={'password'}
                 />
               </InputWrapper>
-              <SubmitButton onClick={submitForm} disabled={!isValid}>
+              <SubmitButton type={'submit'} disabled={!isValid}>
```
Comment (guilhermedecampo) [inline packages/pismo-id/LoginForm.tsx]:
```suggestion
              <SubmitButton type="submit" disabled={!isValid}>
```

File: packages/pismo-id/PismoID.tsx [hunk 2]
```diff
@@ -117,7 +117,15 @@ export const PismoID = (props: PismoIDProps) => {
             }}
           />
         )}
-        {mode === Modes.RECOVERY_SUCCESS && <RecoverySuccess email={email} />}
+        {mode === Modes.RECOVERY_SUCCESS && (
+          <RecoverySuccess
+            goToLogin={() => {
```
Comment (guilhermedecampo) [inline packages/pismo-id/PismoID.tsx]: extract `goToLogin` to function

File: packages/pismo-id/RecoveryForm.tsx [hunk 3]
```diff
@@ -73,14 +73,14 @@ export const RecoveryForm = (props: RecoveryFormProps) => {
                 onChange={handleChange}
               />
             </InputWrapper>
-            <SubmitButton onClick={() => submitForm()} disabled={!isValid}>
+            <SubmitButton type={'submit'} disabled={!isValid}>
```
Comment (guilhermedecampo) [inline packages/pismo-id/RecoveryForm.tsx]:
```suggestion
            <SubmitButton type="submit" disabled={!isValid}>
```

File: packages/pismo-id/RecoveryForm.tsx [hunk 4]
```diff
@@ -73,14 +73,14 @@ export const RecoveryForm = (props: RecoveryFormProps) => {
...
               ENVIAR
             </SubmitButton>
             <LinkWrapper>
               <ArrowBack>&larr;</ArrowBack>
               <Link onClick={() => goToLogin()}>Lembrou?</Link>
```
Comment (guilhermedecampo) [inline packages/pismo-id/RecoveryForm.tsx]:
```suggestion
              <Link onClick={goToLogin}>Lembrou?</Link>
```

File: packages/pismo-id/RecoverySuccess.tsx [hunk 5]
```diff
@@ -1,16 +1,26 @@
...
 }
 
 export const RecoverySuccess = (props: RecoverySuccessProps) => {
-  const { email } = props
+  const { email, goToLogin } = props
   return (
     <FormCard>
       <Title>{`As instruções para redefinição de senha foram enviadas para ${email}`}</Title>
+      <ArrowBack>&larr;</ArrowBack>
+      <Link onClick={() => goToLogin()}>Voltar ao login</Link>
```
Comment (guilhermedecampo) [inline packages/pismo-id/RecoverySuccess.tsx]:
```suggestion
      <Link onClick={goToLogin}>Voltar ao login</Link>
```

File: packages/pismo-id/RecoverySuccess.tsx [hunk 6]
```diff
@@ -1,16 +1,26 @@
...
+  font-size: 1.25rem;
+  color: ${colors.grey800};
+`
+
 interface RecoverySuccessProps {
   email: string
+  goToLogin(): void
 }
 
 export const RecoverySuccess = (props: RecoverySuccessProps) => {
```
Comment (guilhermedecampo) [inline packages/pismo-id/RecoverySuccess.tsx]: can't you still open the props object on the function on typescript and eliminate this `return` and `const` here?

File: packages/pismo-id/RecoverySuccess.tsx (hunk 5 above)
Comment (fakenickels) [reply packages/pismo-id/RecoverySuccess.tsx]: top

File: packages/pismo-id/PismoID.tsx (hunk 2 above)
Comment (fakenickels) [reply packages/pismo-id/PismoID.tsx]: with useCallback?

**Comment by medson10** [comment]
image
image

**Comment by medson10** [comment]
image
image

**Comment by guilhermedecampo** [comment]
Some points to do over here still:

- [ ] change docz to storybooks

**Comment by rodrigopandini** [comment]
Looks ok.
It's still in draft mode?
The others enhances can be for next iterations.

**Comment by guilhermedecampo** [comment]
@medson10 merge here and let's open a new PR for Lingui/i18n and storybook changes

**Pull request merged**